week-5/Homework/mandatory/3-project/js/main.js

```diff
@@ -0,0 +1,62 @@
+// Part 1
+let jumptron  = document.querySelector(".jumbotron");
+let blueBtn = document.querySelector("#blueBtn");
+let orangeBtn = document.querySelector("#orangeBtn");
+let greenBtn = document.querySelector("#greenBtn");
+let dntBtn = document.getElementsByClassName("btn-lrg")[0];
+let vlnBtn = document.getElementsByClassName("btn-lrg")[1];
+
+
+blueBtn.addEventListener("click", function () {
+    jumptron.style.backgroundColor = "#588fbd";
+    dntBtn.style.backgroundColor = "#ffa500";
+    vlnBtn.style.backgroundColor = "black";
+    vlnBtn.style.color = "white";
+})
+
+orangeBtn.addEventListener("click", function () {
+    jumptron.style.backgroundColor = "#f0ad4e";
+    dntBtn.style.backgroundColor = "#5751fd";
+    vlnBtn.style.backgroundColor = "#31b0d5";
+    vlnBtn.style.color = "white";
+})
+
+greenBtn.addEventListener("click", function () {
+    jumptron.style.backgroundColor = "#87ca8a";
+    dntBtn.style.backgroundColor = "black";
+    vlnBtn.style.backgroundColor = "#8c9c08";
```

Comment:
Awesome work! There is an easier way to select both buttons:
```
let dntBtn = document.querySelector(".buttons .btn-primary");
let vlnBtn = document.querySelector(".btn-secondary");
```
It’s more error-proof as the order of array elements returned from `getElementsByClassName` method can change when you add more buttons to the website.
Another thing is that all three event listeners make very similar changes, therefore there is some repetition in your code. Can you think of the way how you could refactor the code so there is only one function responsible for making these changes?